CMakeLists.txt

```diff
@@ -168,6 +168,8 @@ else()
     set(LIB_INSTALL_DIR lib)
 endif()
 
+include_directories(include/)
```

Comment:
@Patater Do you see any incompatibility between this line and getting the proper include path with the crypto submodule?

Reply:
Yes, this shouldn't be necessary to add. Maybe this was readded during a rebase.

See 4cb814e3a780 for why we don't want this line anymore. We used to have this line, but removed it for submodule support.

Reply:
I need include/CMakeLists.txt to be included so I can see the `INSTALL_MBEDTLS_HEADERS` option in `3rdparty/*`. It definitely doesn't work without that line, but I've rearranged it make this a bit cleaner. Not sure what your strategy going forward is for this, so feel free to request changes to that. See 16bdf8c9ea7bf8c7182b22f407fafe3eb892f092.